apps/archive/archive_media.py

```diff
@@ -45,12 +45,12 @@ def on_create(self, docs):
             doc['media'].seek(0)
             file_type = content_type.split('/')[0]
             if file_type == 'video' and app.config.get("VIDEO_SERVER_ENABLE"):
-                if not self.videoEditor.check_video_server():
+                if not self.video_editor.check_video_server():
```

Comment:
Do you think it's necessary to use this method? 
If the server is unavailable (`self.video_editor.check_video_server`) we `raise SuperdeskApiError`, but we do the same in let's say in `self.video_editor.create_timeline_thumbnails`. We raise `SuperdeskApiError` in all `VideoEditorWrapper` methods which do network calls, so it's the same. The only benefit I see in the current approach is a better log message, but we send 2 requests instead of 1. It's not a big deal actually.

Reply:
Hi @ride90, we do this method, because we want if the video server is disconnect it will auto upload to `Superdesk`. But now we think it not necessary, we should raise error when video server enable but its disconnect.
We will remove checking method.